dashboard/settings.py

```diff
@@ -496,6 +493,15 @@ def show_debug_toolbar(request):
 SESSION_COOKIE_SAMESITE = ENV.get("SESSION_COOKIE_SAMESITE", None)
 CSRF_COOKIE_SAMESITE = ENV.get("CSRF_COOKIE_SAMESITE", None)
 
+# Give a chance to enable the backend login via a setting
+ENABLE_BACKEND_LOGIN = ENV.get("ENABLE_BACKEND_LOGIN", ENABLE_BACKEND_LOGIN)
+
```

Comment:
This looks like it would work, but I wonder if instead of re-assigning throughout, you could just handle it all here, doing something like this:
```
ENABLE_BACKEND_LOGIN = True
# would have to create the STUDENT_DASHBOARD_SAML variable
if STUDENT_DASHBOARD_SAML or STUDENT_DASHBOARD_LTI:
   ENABLE_BACKEND_LOGIN = False
# Allow for ENABLE_BACKEND_LOGIN override
 ENABLE_BACKEND_LOGIN = ENV.get("ENABLE_BACKEND_LOGIN", ENABLE_BACKEND_LOGIN)
```

Reply:
Yeah I was trying to think of the best way to do this and re-wrote it a few times. This is needed by the urls.py.

The only thing we'd need to remember is if another auth method got enabled we'd have to add it there. But that would be the same as in the other case too. I can change this.

Reply:
It may not be a huge difference, but I like that the logic for `ENABLE_BACKEND_LOGIN` is all condensed, you don't have to go searching for variable references.

Reply:
How about 
`ENABLE_BACKEND_LOGIN = False if STUDENT_DASHBOARD_SAML or STUDENT_DASHBOARD_LTI else True `

Reply:
STUDENT_DASHBOARD_SAML wasn't set and I noticed the default was True. I'm going to change this to False and have this as a setting.

Reply:
One liner works. For some reason I haven't come around to using the ternary approach in Python as much yet, but use it all the time in JS.